src/Samples/CSharp/Analyzers/CSharpAnalyzers/CSharpAnalyzers.Vsix/project.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "dependencies": {
+    "Microsoft.VisualStudio.Settings.14.0": "14.3.25407",
+    "Microsoft.VSSDK.BuildTools": "14.3.25407"
```

Comment:
This is a Samples project. Ideally we would create a smaller specialized targets file that does just the Dev14/Dev15 switching so that the sample can target either and still be self contained.